src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs

```diff
@@ -471,6 +471,15 @@ internal int ObjectID
 
         }
 
+        /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlBulkCopy.xml' path='docs/members[@name="SqlBulkCopy"]/RowsCopied/*'/>
+        public int RowsCopied
```

Comment:
Not a blocker just an observation, this could be expression form `public int RowsCopied => _rowsCopied;'

Reply:
I kept it same as other places to ensure it's readable enough that it's just the Getter that is available as public API.

Reply:
Fair enough.